### ISSUE TITLE
Searching for EU elections results, no relevant sites

### ISSUE THREAD

**Issue by louwers** [StractOrg/stract#208]
<img width="849" alt="image" src="https://github.com/StractOrg/stract/assets/649392/a93e38a1-0ec8-4e51-a2cb-9f59f88dee59">

<img width="807" alt="image" src="https://github.com/StractOrg/stract/assets/649392/633f5029-f0d0-4c5d-ba4a-eb76a9812ebf">

<img width="812" alt="image" src="https://github.com/StractOrg/stract/assets/649392/e7c93aff-deaf-4d82-ba4d-d48c1c212a0c">

The first link on Google:

- https://results.elections.europa.eu/en/
- https://results.elections.europa.eu/en/national-results-overview/
- https://www.theguardian.com/world/ng-interactive/2024/jun/09/european-elections-results-2024-europe-eu-parliament

**Comment by louwers** [comment]
@julianfairfax No it is not vague. There is only one EU wide election. And even if it were vague, a search engine should be able to deal with vague queries...

**Comment by mikkeldenker** [comment]
I agree the results need to improve here. Fast and accurate ranking is by far the most difficult part of search, so it will take some time to get right.
